cg/apps/housekeeper/hk.py

```diff
@@ -92,7 +94,7 @@ def add_file(self, path, version_obj: models.Version, tags, to_archive=False) ->
 
     def files(
         self, *, bundle: str = None, tags: List[str] = None, version: int = None, path: str = None
-    ) -> Iterable[models.File]:
+    ) -> Query:
```

Comment:
I think this can be done when we retire alchy. I didnt change functionality, just fixed wrong type hint. No need to change functionality now since we are going to retire alchy and do it again later